wordpress-mobile/WordPress-iOS
# Comment Reply: update fullscreen UI

Ref: #17412 

This updates the UI for the fullscreen Reply view.

Notes:
- There is no placeholder text yet.

To test:
- Go to any of these views as they all use the same Reply view:
  - My Site > old Comment details.
  - My Site > new Comment details.
  - Notifications > Comments > notification details.
  - Reader > post comments > reply to post or comment.
- Tap the up arrow to enter fullscreen reply.
- Verify the nav title does not appear.
- Verify the nav bar bottom line does not appear.
- Verify the `Reply` button is disabled.
- Enter some text. Verify the `Reply` button is enabled.

| ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-11-10 at 13 12 29](https://user-images.githubusercontent.com/1816888/141186493-e449f1c8-ecce-4a61-861a-e1fbce173a80.png) | ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-11-10 at 13 12 34](https://user-images.githubusercontent.com/1816888/141186533-2950754c-f9b8-4b2b-be19-be51141f08c6.png) |
|--------|-------|

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Verified the UI looked and functioned correctly.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
